addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -437,6 +437,7 @@ def test_message_process_write_to_catchall(self):
     def test_message_process_write_to_catchall_and_alias(self):
         """ Writing directly to catchall and a valid alias should take alias """
         # Test: no group created, email bounced
+        self.env['ir.config_parameter'].set_param('mail.catchall.domain.strict', True)
```

Comment:
maybe this is not needed

Reply:
I mean, without this line, the test  already works fine.

Reply:
Hmmm yes you're right.

Oh man they did such a mess upstream... :man_facepalming: Let me revert this.